README.md

```diff
@@ -114,6 +114,22 @@ Lego is not (and will never be):
 
 🎈 You can [read the core code](https://github.com/Polight/lego/blob/master/src/lib/Component.js) that is only 61 <abbr title="Lines Of Code">LOC</abbr> resulting that Lego is **as light as 3Kb** for the full bundle!
 
+## Developing and contributing on Lego
+
+Lego codebase is small so navigating through its code should be pretty straightforward.
+
+### Updating the compiler
+
+For any changes within the `src/compiler` folder, you can test it by running it from one of your Lego project. From this project, simply run `bun [path-from-your-project-to-lego-folder]/bin/compile.js` and see the result on your compiled Lego components.
+
+### Updating the library
+
+For any changes within the `src/lib` folder, you can test it by building it with `npm run build` and using files from `dist` folder within one of your Lego project.
+
+### Contributing
+
+Before opening a pull request with your changes, make sure you have built the `dist` package with the `npm run build` command.
```

Comment:
Waiting for the answer on https://github.com/Polight/lego/pull/66#issuecomment-2915507807. If build is only done on deploy, I will update this section.